meson.build

```diff
@@ -1586,11 +1586,21 @@ endif
 rst2pdf = find_program('rst2pdf', required: get_option('pdf-build'))
 features += {'pdf-build': rst2pdf.found()}
 if features['pdf-build']
+    rst2pdf_help = run_command(
```

Comment:
Hmm, this is unfortunate. There is actually a `version` method for meson for finding external programs but it requires 62.0. Looks like that would break the mingw builds (on 61.2).

Reply:
I don't want to fully drop support for older versions of rst2pdf.

To to this with `version` we first have to check `find_program` with `version` and if it fails again without.
The current way seems simpler.

Reply:
No there's actually a method now. As in, `rst2pdf.version()`. I would have preferred using that to build the `command` but we probably can't bump the meson version any time soon because of LTS. So moot point.

Reply:
At least I learned something, thanks!